grails-bootstrap/build.gradle

```diff
@@ -4,6 +4,7 @@ dependencies {
     compile ( "org.codehaus.groovy:groovy-xml:$groovyVersion" )
     compile ( "org.codehaus.groovy:groovy-templates:$groovyVersion" )
     compile "org.yaml:snakeyaml:1.14"
+    compile "io.methvin:directory-watcher:0.3.0" //TODO: Needs JNA to run
```

Comment:
This should be `compileOnly` scope and then added to the `bootRun` classpath by the gradle plugin